Source/WTF/wtf/text/cf/StringConcatenateCF.h

```diff
@@ -50,7 +51,7 @@ inline StringTypeAdapter<CFStringRef>::StringTypeAdapter(CFStringRef string)
 template<> inline void StringTypeAdapter<CFStringRef>::writeTo<LChar>(std::span<LChar> destination) const
 {
     if (m_string)
-        std::memcpy(destination.data(), CFStringGetCStringPtr(m_string, kCFStringEncodingISOLatin1), CFStringGetLength(m_string));
+        memcpySpan(destination, unsafeMakeSpan(CFStringGetCStringPtr(m_string, kCFStringEncodingISOLatin1), CFStringGetLength(m_string)));
```

Comment:
Should the `CFStringGetCStringPtr` wrapped to return a span be a separate function rather than a subexpression? Maybe it can eventually be shared? I guess it's inconvenient in a header like this.